code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -571,102 +575,99 @@
 	resistance_flags = INDESTRUCTIBLE
 	lock_flags = JOB_LOCK
 
+/obj/effect/vendor_bundle
+	var/list/gear_to_spawn = list()
+	var/list/spawned_gear = list()
```

Comment:
autodoc is missing everywhere

Reply:
also use lazylists here please

Reply:
Replaced with lazylists, and added autodoc to the new classes, methods, variables in PR